packages/ui/src/components/timeline/Label.tsx

```diff
@@ -59,6 +60,18 @@ export function Label({event, scene}: LabelProps) {
             handleDrag(e);
           }
         }}
+        onPointerDown={event => {
+          event.currentTarget.setPointerCapture(event.pointerId);
+        }}
+        onPointerMove={event => {
+          if (event.currentTarget.hasPointerCapture(event.pointerId)) {
+            labelClipDraggingSignal.value = {left: eventTime, dragging: true};
+          }
+        }}
+        onPointerUp={event => {
+          event.currentTarget.releasePointerCapture(event.pointerId);
+          labelClipDraggingSignal.value = {left: null, dragging: false};
+        }}
```

Comment:
You didn't remove the old `useDrag`.
It should pretty straightforward, just move the first callback from `useDrag` to `onPointerMove` and the second one to `onPointerUp`.
Same goes for `onMouseDown`, there's no need to have both it and `onPointerDown`.

Reply:
Moved everything to `pointer events` removing the old `useDrag hook` and `mouse event (onMouseDown)`